lib/wanda/executions/execution.ex

```diff
@@ -20,10 +19,7 @@ defmodule Wanda.Executions.Execution do
     field :result, :map, default: %{}
     field :status, Ecto.Enum, values: [:running, :completed]
 
-    embeds_many :targets, Target do
-      field :agent_id, Ecto.UUID
-      field :checks, {:array, :string}
-    end
+    embeds_many :targets, Wanda.Executions.Execution.Target
```

Comment:
The inline `embeds_many` doesn't allow to add the json derive..
And i couldn't make it work without extracting it to a independent module